site/content/en/contribute/howitworks/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Writing Code"
+linkTitle: "Writing Code"
+type: docs
+weight: 40
+description: >
+    How to modify Kustomize
+---
+
+{{< alert color="success" title="Info" >}}
+Kustomize supports go workspace mode, so if adding locally modified modules, remember to also add it to the [go.work](https://github.com/kubernetes-sigs/kustomize/blob/master/go.work) file.
```

Comment:
added this comment [as per natasha's feedback](https://github.com/kubernetes-sigs/kustomize/pull/4709#discussion_r922549656)